.settings/org.eclipse.buildship.core.prefs

```diff
@@ -1,13 +1,13 @@
-arguments=
+arguments=--init-script /var/folders/jz/4rf1r3wx21g62r8w_gd7c2kh0000gp/T/db3b08fc4a9ef609cb16b96b200fa13e563f396e9bb1ed0905fdab7bc3bc513b.gradle --init-script /var/folders/jz/4rf1r3wx21g62r8w_gd7c2kh0000gp/T/52cde0cfcf3e28b8b7510e992210d9614505e0911af0c190bd590d7158574963.gradle
 auto.sync=false
 build.scans.enabled=false
 connection.gradle.distribution=GRADLE_DISTRIBUTION(WRAPPER)
 connection.project.dir=
 eclipse.preferences.version=1
 gradle.user.home=
-java.home=
+java.home=/Users/darogers/java17_macosx_x86_64/jre/Contents/Home
```

Comment:
We can't have `/Users/darogers` hardcoded in a prefs file?!?

This obviously will not work on any other's PC than the one owned by Daymon!!